src/ResourceManager/KeyVault/Commands.KeyVault/Commands/BackupAzureKeyVaultKey.cs

```diff
@@ -12,92 +12,129 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.KeyVault.Models;
 using System;
-using System.IO;
 using System.Management.Automation;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.KeyVault.Models;
 using KeyVaultProperties = Microsoft.Azure.Commands.KeyVault.Properties;
 
 namespace Microsoft.Azure.Commands.KeyVault
 {
     /// <summary>
     /// Requests that a backup of the specified key be downloaded and stored to a file
     /// </summary>
+    /// <remarks>
+    /// The cmdlet returns the path of the newly created backup file.
+    /// </remarks>
     [Cmdlet(VerbsData.Backup, "AzureKeyVaultKey",
         SupportsShouldProcess = true, 
         HelpUri = Constants.KeyVaultHelpUri)]
```

Comment:
Need to add a DefaultParameterSetName value, since you now have multiple parameter sets